test/Test/Analysis.hs

```diff
@@ -0,0 +1,14 @@
+module Test.Analysis
+    ( analysisSpecs
+    ) where
+
+import HieTypes (HieFile (..))
+import Test.Hspec (Spec, describe, it, shouldBe)
+
+import Stan.Hie (countLinesOfCode)
+
+
+analysisSpecs :: HieFile -> Spec
+analysisSpecs hieFile = describe "Analysis tests" $
+    it "should count lines of code of example file" $
```

Comment:
What do you think about adding a test like `analysisModules > 10`? The number of modules will change all the time, but maybe we can at least implement some sanity check to make sure that the number of modules is calculated?

Reply:
Oh, interesting idea!
I would even prefer to have the equality check on modules, given that soon it will stabilise.
The thing is that currently there are no `.hie` file for `Spec.hs` module of tests, and I don't know the reason. This looks a bit unstable. But I think that it is already a nice sanity check — to increment the number in tests when you add another module in the project.

Reply:
This also sounds reasonable :+1: Absence of the `.hie` file for `Spec.hs` sounds like a GHC bug...